ultron-compose/src/androidMain/kotlin/com/atiurin/ultron/core/compose/ComposeRuleContainer.android.kt

```diff
@@ -58,6 +60,13 @@ inline fun <reified A : ComponentActivity> createUltronComposeRule(): AndroidCom
     return rule
 }
 
```

Comment:
Lets add some documentation about `createSimpleUltronComposeRule ` this method to [rules creation with compose](https://github.com/open-tool/ultron/blob/master/docs/docs/compose/android.md) to clarify then this method should be used.